plugins/org.eclipse.elk.alg.graphviz.dot/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -8,8 +8,15 @@ org.eclipse.jdt.core.compiler.debug.lineNumber=generate
 org.eclipse.jdt.core.compiler.debug.localVariable=generate
 org.eclipse.jdt.core.compiler.debug.sourceFile=generate
 org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
+org.eclipse.jdt.core.compiler.problem.enablePreviewFeatures=disabled
 org.eclipse.jdt.core.compiler.problem.enumIdentifier=error
+<<<<<<< HEAD
 org.eclipse.jdt.core.compiler.source=1.8
+=======
+org.eclipse.jdt.core.compiler.problem.reportPreviewFeatures=warning
```

Comment:
Looks like there are some merge conflict markers here as well that were not resolved